apps/ledger-live-desktop/src/renderer/families/canton/PendingTransferProposals/index.tsx

```diff
@@ -136,7 +138,7 @@ const PendingTransferProposals: React.FC<Props> = ({ account }) => {
   const handleRowClick = useCallback(
     (contractId: string) => {
       setDrawer(PendingTransferProposalsDetails, {
-        account,
+        account: account,
```

Comment:
Sorry, can you explain why you need to assign it ? I may miss something, but it should work like this 
```suggestion
        account,
```